### PR TITLE
Updated README

### LINKED ISSUE #1203: The README has nothing to be read
As this is an open repository, with a hope and desire to grow, the `README` should at least contain some bare minimum information on where to get our version of the compiler and related ooc tools.

This link has a lot of good ideas for the contents of a readme: http://stackoverflow.com/questions/2304863/how-to-write-a-good-readme

### PR DESCRIPTION
**Pull request by marcusnaslund**
Obviously not ready yet, but this eventually fixes #1203 

Thoughts, ideas, complaints @thomasfanell @fredrikbryntesson @simonmika ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
-ooc port of parts of Kean
+This is an extended SDK for `magic`, a high-level cross-platform language which transpiles into C99.
 
-Uses [Sean Barrett's stb library](https://github.com/nothings/stb) for opening and saving images.
+## History
+This project was originally a fork of the [ooc project](https://github.com/fasterthanlime/rock) and contains ports of selected parts of the C# library [Kean](https://github.com/cogneco/Kean). The most notable changes include dropping garbage collection. It also uses [Sean Barrett's stb library](https://github.com/nothings/stb) for opening and saving images.
+
+## How to run
+To compile, you need the Cogneco version of the [Rock compiler](https://github.com/cogneco/rock). To make life easy, you might also want an editor with ooc/magic syntax highlighting support and a style enforcer. Those are available (for Ubuntu) at the [magic-tools repo](https://github.com/cogneco/magic-tools#installation-ubuntu), which can also be used to automatically install (and update) rock.
+
```
Comment (fredrikbryntesson) [inline README.md]: https://github.com/cogneco/magic-tools#installation-ubuntu should now be https://github.com/magic-lang/magic-tools#installation-ubuntu

Comment (fredrikbryntesson) [inline README.md]: And https://github.com/cogneco/rock is https://github.com/magic-lang/rock

File: README.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+## How to learn magic
+There are several original sources:
+
+- [The original ooc documentation](https://ooc-lang.org/docs/)
+- [The original ooc tutorial](https://ooc-lang.org/docs/tutorial/)
+- [Wikibooks](https://en.wikibooks.org/wiki/Programming_with_ooc)
+
+The above however target the original ooc language and are as such getting more and more outdated. A slightly more updated version is available [here](http://oocmanual.cogneco.com/language/tuples/). **TODO**
+
```
Comment (fredrikbryntesson) [inline README.md]: Manual may not be hosted on cogneco.com in the future so maybe we should have a link directly to the Github repository.

Comment (marcusnaslund) [inline README.md]: @jonathanudd and I will develop a new site for magic which I will link to later.

File: README.md (hunk 1 above)
Comment (marcusnaslund) [inline README.md]: Thanks

**Comment by marcusnaslund** [comment]
There we go, should be enough for now. Peer review @thomasfanell ?

**Comment by marcusnaslund** [comment]
Or @fredrikbryntesson ?

**Pull request merged**